docs/source/installation/plugins.rst

```diff
@@ -0,0 +1,72 @@
+.. _plugins:
+
+=======
+Plugins
+=======
+
+Plugins are features that extend Manim's core functionality. Since Manim is
+extensible and not everything belongs in its core, we'll go over how to
+install, use, and create your own plugins.
+
+*Note: The standard naming convention for plugins is to prefix the plugin*
+*with* ``manim-``
+
+Installing Plugins
+******************
+Plugins can be easily installed to your python site-packages via the ``pip``
```

Comment:
Suggestion: manim has many users who are not python experts, so perhaps this is more accessible:
```suggestion
Plugins can be easily installed via the ``pip``
```